configure.ac

```diff
@@ -2569,8 +2523,6 @@ SURICATA_BUILD_CONF="Suricata Configuration:
 
   Python support:                          ${enable_python}
   Python path:                             ${python_path}
-  Python distutils                         ${have_python_distutils}
-  Python yaml                              ${have_python_yaml}
```

Comment:
Q: Why do we remove checks for yaml?

Reply:
`suricatasc` and `suricatactl` do no require it.  And `suricata-update` will gracefully error out and say
```
error: pyyaml is required
```
I like the idea that the user can install Suricata, try to run Suricata-Update then simply install pyyaml and carry on.  Rather than installing pyyaml then having to re-install Suricata.